xlators/features/marker/src/marker-quota.c

```diff
@@ -1685,9 +1685,7 @@ mq_initiate_quota_task(void *opaque)
             goto out;
         }
 
-        if (prev_dirty == 0) {
-            ret = mq_mark_dirty(this, &parent_loc, 0);
```

Comment:
We still need to cover the case when prev_dirty is zero. What will happen if this condition is true.